src/utils.test.js

```diff
@@ -323,6 +323,46 @@ describe('utils', () => {
     process.env['LUMIGO_VALID_ALIASES'] = undefined;
   });
 
+  test('getTimeoutTimerBuffer -> ENV_VAR', () => {
+    process.env['LUMIGO_TIMEOUT_BUFFER'] = '0.35';
+    expect(utils.getTimeoutTimerBuffer()).toEqual(350);
+  });
+
+  test('getTimeoutTimerBuffer -> ENV_VAR ms', () => {
+    process.env['LUMIGO_TIMEOUT_BUFFER_MS'] = '350';
+    expect(utils.getTimeoutTimerBuffer()).toEqual(350);
+  });
+
+  test('getTimeoutTimerBuffer -> Min value', () => {
+    expect(utils.isValidAlias()).toEqual(true);
+    TracerGlobals.setHandlerInputs({
+      context: {
+        getRemainingTimeInMillis: () => 1000,
```

Comment:
Just to make sure: you have something that limit the TO mechanism to not work on these short lambdas, right?

Reply:
Only if the getRemainingTimeInMillis < Timeout buffer. 
What the minimum should be ?

Reply:
That's a problem because if the lambda run 600ms, then the TO will jump 100ms after it started... :(
In the python tracer we added a minimum of 2s to the remaining time

Reply:
Added